src/explore-education-statistics-frontend/src/services/redirectService.ts

```diff
@@ -0,0 +1,21 @@
+export interface Redirects {
+  methodologyRedirects: Redirect[];
```

Comment:
Again, given that these attributes are within something called `Redirects`, can they just be plain `methodologies` and `publications`?  If you think the readability's better the way it is though, I've not got a strong opinion on it.